src/com/company/reverse/Reverse.java

```diff
@@ -2,11 +2,14 @@
 
```

Comment:
please follow default project structure 
```
src/
    main/
      java/
    test/
      java/
```

Reply:
in this case please put main code to `main` directory

Reply:
Done